gh-combine-prs

```diff
@@ -24,7 +24,9 @@ Usage: gh combine-prs --query "QUERY"
 		--skip-pr-check
 				if set, will combine matching PRs even if they are not passing checks.
 				Defaults to false when not specified
-
+		--non-interactive
+				if set, will not prompt for confirmation before combining PRs. Warning: Can be destructive as we force push to achieve this.
```

Comment:
```suggestion
				if set, will not prompt for confirmation before combining PRs. Warning: Can be destructive as we force push to combined-pr-branch to achieve this.
```
This makes it a bit clearer, and a bit less scary about exactly which branch will be force-pushed.
WDYT?